backend/core/src/shared/types/geocoding-values.ts

```diff
@@ -0,0 +1,5 @@
+export enum GeocodingValues {
```

Comment:
Curious why we're using this values instead of true / false / unknown?

Reply:
Good question! They are getting saved into the DB as a string but there was no reason I chose "yes"/"no" instead of "true"/"false"